pkg/sql/scheduledlogging/TestCaptureIndexUsageStats is flaky

Failed on my PR with,

```
Error:        "1s" is not greater than "1.150076s"
          Test:         TestCaptureIndexUsageStats
          Messages:     1s <= 3.150076s-2s
    panic.go:522: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/69cc8b5ff8b35dcd9569961cf48105ef/logTestCaptureIndexUsageStats249550260
--- FAIL: TestCaptureIndexUsageStats (8.50s)
```

TC says this test started failing today with 14 failures seen; most recent failure: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/7550776


Jira issue: CRDB-21520

--- a/pkg/sql/scheduledlogging/captured_index_usage_stats_test.go
+++ b/pkg/sql/scheduledlogging/captured_index_usage_stats_test.go
@@ -79,7 +79,7 @@ func TestCaptureIndexUsageStats(t *testing.T) {
 	stubLoggingDelay := 0 * time.Second
 
 	// timeBuffer is a short time buffer to account for non-determinism in the logging timings.
-	const timeBuffer = 2 * time.Second
+	const timeBuffer = 3 * time.Second
 
 	settings := cluster.MakeTestingClusterSettings()
 	// Configure capture index usage statistics to be disabled. This is to test

